packages/store/types/IStore.js

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
Although these are empty, they seem to be required to get e.g. next.js' build pipeline working. Otherwise you get errors like
```
packages/app dev: error - ../contracts/types/index.ts
packages/app dev: Module parse failed: Unexpected token (4:12)
packages/app dev: You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
packages/app dev: | /* tslint:disable */
packages/app dev: | /* eslint-disable */
packages/app dev: > import type * as afdSol from "./AFD.sol";
```